### PR TITLE
Update user manual for CCR and pSCR dive logs

### PR DESCRIPTION
**Pull request by willemferguson**
Update the text pertaining to CCR and pSCR dive logs to include the
changes with respect to:
1) Colour of CCR setpoint graph and pSCR OC-equivalen pO2 graph.
2) Display of bailout information on dive logs.
3) Preferences tab allowing setting pertaining to the above.

This is the first of a few PRs with updates to the user manual
relating to CCR and pSCR dives.

Signed-off-by: Willem Ferguson <willemferguson@zoology.up.ac.za>

### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [x ] Documentation change
- [ ] Language translation

### CODE REVIEW

File: Documentation/user-manual.txt [hunk 1]
```diff
@@ -2033,14 +2033,46 @@ To log pSCR dives, no special procedures are required. Use the normal steps outl
...
+in the loop can be visualised in _Subsurface_ by opening the _Profile_
+panel in the _File -> Preferences_ menu item and then providing suitable information in the section
+labelled _pSCR options_. The _pSCR metabolic rate_ is your rate of oxygen metatolism in liters/min (commonly
+between 0.5 and 2.5). The _Dilution ratio_ is the ratio indicating how much gas in the loop is replaced with
+fresh gas, relative to the gas being breathed (common values are between 1:3 and 1:10). These values are also
+used when planning pSCR dives. If the oxygen content of the loop is not recorded by an oxygen sensor, _Subsurface_
+estimates the pO~2~ in the loop using the information supplied above. This pO~2~ is shown as the green line
+representing the oxygen partial pressure in the dive profile. If the oxygen pressure in the loop is measured using
+an oxygen sensor connected to a dive computer, this information is probably recorded by the dive computer and these
+measured pO~2~ vaues are shown on the dive profile. In addition, the _Preferences_ panel allows one to activate
```
Comment (neolit123) [inline Documentation/user-manual.txt]: s/vaues/values/

File: Documentation/user-manual.txt [hunk 2]
```diff
@@ -2143,7 +2175,41 @@ _Events_: Several events are logged, e.g. switching the mouthpiece to open circu
...
+_Bailout_: Bailout events during a CCR dive can be recorded in a _Subsurface_ divelog. Right-click
+on the dive profile to generate a dive profile context menu. Select _Change divemode_ and choose
+the appropriate bailout option (probably OC). An appropriate icon is shown on the depth profile (see
+image below). In most CCR equipment it is also
+possible to bailout to pSCR, especially in cases where the electronics died and the diluent supply
+is limited. This can also be reflected in the dive log and Subsurface will calculate and graph the
+appropriate pO~2~ values. The case during CCR training is more complex if the instructor prefers
+a low setpoint such as 0.7 to be maintained during bailout. In this case the CCR equipment
+maintains the low setpoint and overrides the OC or pSCR conditions. In such cases it is probably
+more accurate not to record a bailout event in the dive log since _Subsurface_ will calulate pO~2~
```
Comment (neolit123) [inline Documentation/user-manual.txt]: `s/calulate/calculate`

File: Documentation/user-manual.txt [hunk 3]
```diff
@@ -2157,22 +2223,8 @@ _Equipment-specific information_: Equipment-specific information gathered by
...
-
-The default color of the computer-generated deco ceiling is white. This can be set to red
-by checking the appropriate check box after selecting _File  ->  Preferences  ->  Profile_.
-Below is a dive profile indicating the dive computer-generated deco ceiling:
-
-image::images/CCR_dive_ceilingF22.jpg["FIGURE: CCR computer-generated deco ceiling",align="center"]
-
-More equipment-specific information for downloading CCR dive logs for
-Poseidon MkVI and APD equipment can be found in xref:_appendix_b_dive_computer_specific_information_for_importing_dive_information[Appendix B].
+More equipment-specific procdures and information for downloading CCR dive logs for
```
Comment (neolit123) [inline Documentation/user-manual.txt]: `s/procdures/procedures`

**Review by neolit123 — CHANGES_REQUESTED** [review]
thanks for the update.
copied the text changes into a word processor and found a couple of typos.

**Comment by dirkhh** [comment]
@neolit123 this seems to address your concerns? Would you approve and merge?

**Comment by neolit123** [comment]
thanks, @willemferguson .

**Pull request merged**